src/DataFixtures/ORM/LoadCitizenProjectCommentData.php

```diff
@@ -16,14 +16,16 @@ public function load(ObjectManager $manager)
         $comment1 = $this->createCitizenProjectComment(
             $this->getReference('citizen-project-1'),
             $this->getReference('adherent-2'),
-            'Jean-Paul à Maurice : tout va bien ! Je répète ! Tout va bien !'
+            'Jean-Paul à Maurice : tout va bien ! Je répète ! Tout va bien !',
+            '-1 minute'
```

Comment:
Yay on this, it will prevent our "random" failing test, caused by a race condition IMO.

Reply:
Yes, I added this to fix this "random" failing.